cli/triggers.md

```diff
@@ -0,0 +1,112 @@
+# Lamba Triggers
+
+Certain AWS Services can [invoke Lambda functions](https://docs.aws.amazon.com/lambda/latest/dg/lambda-services.html) in response to lifecycle events.  The Amplify CLI provides trigger templates for common use cases.
+
+If you wish to modify the functionality of these templates, you are able to do so locally before pushing them.
+
+## Auth
+
```

Comment:
I think we need to show a code snippet of how customers surface these templates - e.g. `amplify add auth` then select `configure the custom stuff` etc...